opal_common/security/sslcontext.py

```diff
@@ -18,7 +18,7 @@ def get_custom_ssl_context() -> Optional[ssl.SSLContext]:
     if ca_file is None:
         return None
 
-    if len(ca_file) == 0:
+    if ca_file:
```

Comment:
You are correct, I missed it, but rethinking about it, the below code can be replaced with one single check, since both None and len0 list will be evaluated to `False` 
From:
```
if ca_file is None:
        return None
if not ca_file:
        return None
```

To :
```
if not ca_file:
    return None
```